config/vufind/config.ini

```diff
@@ -1308,6 +1310,14 @@ show_in_results = true      ; include in search results
 show_in_record = false      ; include in core record metadata
 show_in_holdings = false    ; include in holdings tab of record view
 
+; Whether to load any third-party icons for the DOI services via VuFind's cover
+; loader proxy to avoid any privacy implications. Ensure that the necessary domains
+; are allowed in Content/coverproxyCache[] setting. Default is false.
+proxy_icons = true
```

Comment:
This says default is false, but it's set to true. Is that intentional? I'm not opposed to having default configuration that differs from default behavior when configuration is disabled, but that can be a bit confusing!

Reply:
I changed default to false so that current behavior is kept.